website/pages/docs/nia/installation/configuration.mdx

```diff
@@ -8,4 +8,257 @@ description: >-
 
 # Configuration Options for Consul-Terraform-Sync
 
-All configuration options are available [here](https://github.com/hashicorp/consul-terraform-sync/blob/master/docs/config.md#consul-terraform-sync).
+The Consul-Terraform-Sync daemon is configured using configuration files and supports [HashiCorp Configuration Language](https://github.com/hashicorp/hcl) (HCL) and JSON file formats.
+
+## Example
+
+An example HCL configuration is shown below to automate a task to execute a Terraform module for 2 services.
+
+```hcl
+log_level = "info"
+
+consul {
+  address = "consul.example.com"
+}
+
+buffer_period {
+  min = "5s"
+  max = "20s"
+}
+
+task {
+  name = "website-x"
+  description = "automate services for website-x"
+  source = "namespace/example/module"
+  version = "1.0.0"
+  providers = ["myprovider"]
+  services = ["web", "api"]
+  wait {
+    min = "10s"
+  }
+}
+
+driver "terraform" {
+  required_providers {
+    myprovider = {
+      source = "namespace/myprovider"
+      version = "1.3.0"
+    }
+  }
+}
+
+provider "myprovider" {
+  address = "myprovider.example.com"
+}
+```
+
+## Consul-Terraform-Sync
```

Comment:
This having just the title of the project seems slightly confusing. Maybe "Global config options" or something along those lines?

Reply:
I honestly could not figure out a decent title for the longest time. "Global config options" is a better fit, thanks for the suggestion